src/icons/cover_art_unknown.svg

```diff
@@ -1,55 +1 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-
-<svg
-   xmlns:dc="http://purl.org/dc/elements/1.1/"
-   xmlns:cc="http://creativecommons.org/ns#"
-   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
-   xmlns:svg="http://www.w3.org/2000/svg"
-   xmlns="http://www.w3.org/2000/svg"
-   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
-   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
-   version="1.1"
-   id="Layer_1"
-   x="0px"
-   y="0px"
-   viewBox="-278 401.9 38 38"
-   style="enable-background:new -278 401.9 38 38;"
-   xml:space="preserve"
-   inkscape:version="0.91 r13725"
-   sodipodi:docname="question.svg"><metadata
-     id="metadata4147"><rdf:RDF><cc:Work
-         rdf:about=""><dc:format>image/svg+xml</dc:format><dc:type
-           rdf:resource="http://purl.org/dc/dcmitype/StillImage" /></cc:Work></rdf:RDF></metadata><defs
-     id="defs4145" /><sodipodi:namedview
-     pagecolor="#ffffff"
-     bordercolor="#666666"
-     borderopacity="1"
-     objecttolerance="10"
-     gridtolerance="10"
-     guidetolerance="10"
-     inkscape:pageopacity="0"
```

Comment:
What was the reason for optimising these svgs? Should we maintain this meta data?

Reply:
Like _nearly_ everything "less is more"... less to download each time is better for everyone and the environment.

Lots of this meta data is just marketing fluff - the images are all so basic that any vector/svg reading app can re-open them and edit them (... should you not like the style of this question mark).

But these icons are "never" re-edited... they are sometimes "replaced". There's no need to add ~75% extra on most of the largest assets in this project for very little reason.